staff/templates/staff_base.html

```diff
@@ -1,82 +1,211 @@
+            $(this).removeClass("deintro").addClass("intro");
+        }
+      });
+    });
 
-{% endblock %}
+    $(document).ready(function(){
+      var $rightpanel = $('#mui-panel');
+      $(".instafeedtrigger").click(function() {
+        if ($rightpanel.hasClass('rightpanelbody-hidden')) {
+            $rightpanel.removeClass('rightpanelbody-hidden').addClass('rightpanelbody-visible');
+        } else if ($rightpanel.hasClass('rightpanelbody-visible')){
+            $rightpanel.removeClass('rightpanelbody-visible').addClass('rightpanelbody-hidden');
+        };
+      });
+    });
+    </script>
+</head>
+<body class="hide-sidedrawer">
+    <header id="header" style="box-shadow: 0px 3px 35px 0px gray">
+      <div class="mui-appbar mui--appbar-line-height">
+        <div class="mui-container-fluid">
+          <a class="sidedrawer-toggle mui--visible-xs-inline-block mui--visible-sm-inline-block js-show-sidedrawer">☰</a>
+          <a class="sidedrawer-toggle mui--hidden-xs mui--hidden-sm js-hide-sidedrawer">☰</a>
+           <div style="float: right; cursor: pointer;" class="mui-dropdown mui-dropdown--left">
+              <a class="mui--text-left" data-mui-toggle="dropdown">
+                <i class="material-icons" style="color: white">more_vert</i>
+                <span class="mui-caret--left"></span>
+              </a>
+              <ul class="mui-dropdown__menu">
+                <li><a style="cursor: pointer;" class="instafeedtrigger">Instafeed</a></li>
+                <li><a href="{% url 'home-logout' %}">Log out</a></li>
+              </ul>
+          </div>
+        </div>
+        </div>
+      </div>
+    </header>
+    <div id="mui-panel" class="rightpanel rightpanelbody-hidden">
+      <h2 style="text-align: center; padding-bottom: 10px; font-size: 35px; color: #474747;">Instagram Feed</h2><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.2" />
+      <div id="instafeed">
+        <!-- INSTAFEED STUFF DOWN HERE !!!DO NOT CODE DOWN BELOW!!! -->
+      </div>
+    </div>
+    <div id="content-wrapper">
+      <div class="mui--appbar-height"></div>
+      <div class="mui-container-fluid">
+        <br>
+        {% block content %}
 
+        {% endblock %}
+      </div>
+    </div>
+    <footer id="footer" style="background-color: #5c5c5c; text-align: center">
+      <div class="mui-container-fluid">
+        <!-- Links down below -->
+        <br>
+        <div style="text-align: center; vertical-align: text-bottom">
+            <br>
+            <br>
+        </div>
+      </div>
+      <h4>© Copyright 2019</h4>
+    </footer>
 
-</body>
+    <div id="sidedrawer" class="mui--no-user-select">
+          <div id="sidedrawer-brand" class="mui--appbar-line-height">
+          </div>
+          <div class="mui-divider"></div>
+            <br>
+            <img src="{% static 'missionbit-logo.png' %}" style="width: 80%; padding-left: 10%">
```

Comment:
An SVG version would look better especially since it's getting resized

Reply:
Resolved in 9dcd3c22296acf8f3d38f201b6ad367c32be67a0